README.md

```diff
@@ -47,7 +47,7 @@ This is a tool meant for VIM users. In other words, it is not a VIM plugin
 
 The only hard dependency is [SyntaxRange](https://github.com/vim-scripts/SyntaxRange).
 
-Other than that you may need some syntax highlighting plugins to handle the code syntax highlighting.
+Other than that you may need some syntax highlighting plugins to handle the code syntax highlighting. [vim-scripts/ingo-library](https://github.com/vim-scripts/ingo-library) may be required for syntax highlighting.
```

Comment:
As far as I can tell, ingo-library is a dependency of SyntaxRange -
here's a stack trace of what happens if you try without ingo-library:

"presentation/slide003.md" 115L, 1465C
Error detected while processing function SyntaxRange#SyntaxInclude:
line    1:
E117: Unknown function: ingo#range#NetStart
E15: Invalid expression: [ingo#range#NetStart(a:startLnum), ingo#range#NetEnd(a:endLnum)]
line    2:
E121: Undefined variable: l:startLnum
E116: Invalid arguments for function printf('\%%%dl', l:startLnum),   (l:startLnum < l:endLnum && l:endLnum == line('$') ?       '\%$' :       printf('
\%%%dl', (l:endLnum + 1))   ),   a:filetype)
"presentation/slide004.md" 111L, 1379C
line    1:
E117: Unknown function: ingo#range#NetStart
E15: Invalid expression: [ingo#range#NetStart(a:startLnum), ingo#range#NetEnd(a:endLnum)]
line    2:
E121: Undefined variable: l:startLnum
E116: Invalid arguments for function printf('\%%%dl', l:startLnum),   (l:startLnum < l:endLnum && l:endLnum == line('$') ?       '\%$' :       printf('
\%%%dl', (l:endLnum + 1))   ),   a:filetype)
E116: Invalid arguments for function SyntaxRange#Include


On Sat, Jun 13, 2015 at 02:34:23PM -0700, Tyler Benziger wrote:
> > @@ -47,7 +47,7 @@ This is a tool meant for VIM users. In other words, it is not a VIM plugin
> >  
> >  The only hard dependency is [SyntaxRange](https://github.com/vim-scripts/SyntaxRange).
> >  
> > -Other than that you may need some syntax highlighting plugins to handle the code syntax highlighting.
> > +Other than that you may need some syntax highlighting plugins to handle the code syntax highlighting. [vim-scripts/ingo-library](https://github.com/vim-scripts/ingo-library) may be required for syntax highlighting.
> 
> There are many other plugins that do Markdown syntax highlighting. https://github.com/tpope/vim-markdown & https://github.com/plasticboy/vim-markdown are just two other examples. I don't know much about ingo-library, but I do know that recommending finding *any* markdown syntax highlighting plugin is safer than recommending a specific one. I'd be happy to merge this in if this part were taken out.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/tybenz/vimdeck/pull/35/files#r32374107

-- 
Kris